instaclone/src/components/PostContainer/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './CommentSection.css';
+import ActualComment from './ActualComment';
+
+class CommentSection extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			comments: this.props.comment,
+			newComment: '',
+			likes: 0
+		};
+	}
+	commentInput = e => {
+		this.setState({ newComment: e.target.value });
+	};
+
+	addNewComment = event => {
+		event.preventDefault();
+		const userName = 'Michael';
+		const nComment = { text: this.state.newComment, username: userName };
+		const commentList = this.state.comments.slice();
+		commentList.push(nComment);
+		this.setState({
+			comments: commentList
+		});
+		event.target.reset();
+	};
+
+	addLike = event => {
+		let newLikes = this.state.likes;
+		++newLikes;
```

Comment:
Be sure you know the differences between prefixing/postfixing ++. Check out this [quick 2-min](https://codeburst.io/javascript-increment-and-decrement-8c223858d5ed) read on the differences.